installer/algorand@.service.template-user

```diff
@@ -0,0 +1,28 @@
+## This file should be installed as:
+##
+##   /home/$USER/.config/systemd/user/algorand@.service
+##
+## and then running "systemctl --user daemon-reload".
+##
+## To enable and start algod for a particular directory, run:
+##
+##   systemctl --user enable algorand@$(systemd-escape /home/algo/algorand/testnet/data/relay)
+##   systemctl --user start  algorand@$(systemd-escape /home/algo/algorand/testnet/data/relay)
+##
+## This assumes a directory layout where the algod binary lives at
+## /home/algo/algorand/testnet/algod, and the data lives under
+## /home/algo/algorand/testnet/data/relay.
+
+[Unit]
+Description=Algorand daemon under %I
+After=network.target
+AssertPathExists=%I
+
+[Service]
+ExecStart=/bin/bash -c "exec /%I/../algod -d %I"
```

Comment:
It's awfully fragile that the bin dir must be relative to the data dir. Is that documented? Can we fix that and substitute in the bin dir when installing this in systemd-setup ? (e.g. use `sed` instead of `cp`). Then we could drop the `bash` layer to and just have `ExecStart=/path/to/algod -d %I`

Reply:
I agree that it's fragile.  I'll look into this as you suggested.

I do have some new docs here: https://github.com/algorand/docs/pull/279

Reply:
No, it's not documented.